src/structures/interfaces/InteractionResponses.js

```diff
@@ -63,7 +63,7 @@ class InteractionResponses {
    * const embed = new MessageEmbed().setDescription('Pong!');
    *
    * interaction.reply({ embeds: [embed] })
-   *   .then(console.log)
+   *   .then('Reply sent.')
```

Comment:
This will throw a big error...
```suggestion
   *   .then(() => console.log('Reply sent.'))
```